examples/spec/twilio_flex_v1.yaml

```diff
@@ -287,7 +287,7 @@ paths:
     x-twilio:
       parent: /Credentials
       pathType: instance
-  /v1/Credentials/AWS/{Sid}/History:
+  /v1/Credentials/AWS/{Sid}/History/{TestInteger}:
     description: A nested instance resource under an instance resource
```

Comment:
This was by-design as there are certain APIs we need to support that match the original pattern.